src/classes/CalculatorClass.js

```diff
@@ -0,0 +1,52 @@
+export default class Calculator {
+  constructor(calculatedNumber, isValidNumber) {
+    this.calculatedNumber = calculatedNumber;
+    this.isValidNumber = isValidNumber;
+  }
+
+  addNumber(number1, number2) {
+    this.isValidNumber = this.validateNumberLength(number1, number2);
+
+    if (this.isValidNumber) {
+      this.calculatedNumber = Math.floor(number1 + number2);
+      return this.calculatedNumber;
+    }
+  }
```

Comment:
```suggestion
  addNumber(number1, number2) {
    this.validateNumberLength(number1, number2);
    return Math.floor(number1 + number2);
  }
```
앞선 리뷰를 토대로, 이렇게 표현해볼 수 있지 않을까 싶네요!

다만 이럴 경우 지금 쓰이는 멤버변수들(isValidNumber, calculatedNumber 등)이 의미가 있는걸까요?